src/include/duckdb/common/vector_operations/aggregate_executor.hpp

```diff
@@ -17,13 +17,18 @@ namespace duckdb {
 
 // structs
 struct AggregateInputData;
+
+// The bounds of a window frame
 struct FrameBounds {
 	FrameBounds() : start(0), end(0) {};
 	FrameBounds(idx_t start, idx_t end) : start(start), end(end) {};
 	idx_t start = 0;
 	idx_t end = 0;
 };
 
+// A set of window subframes for windowed EXCLUDE
+using SubFrames = vector<FrameBounds>;
```

Comment:
Before looking at this file I was wondering about the name `SubFrames`. Can we rename this to something like `ExcludedFrames`?

Reply:
Well they are actually _included_ frames, so that's probably not a good idea. I used to call them excluded frames but Adrian and I started calling them "subframes" so I think that is now the term of art(!)

Reply:
Okay makes sense, then!